webservices/schemas.py

```diff
@@ -387,6 +387,17 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'], schemas['Ca
 )
 augment_schemas(FilingsSchema)
 
+EFilingsSchema = make_schema(
+    models.EFilings,
+    fields={
+        'beginning_image_number': ma.fields.Str(),
+        'ending_image_number': ma.fields.Str(),
+        'pdf_url': ma.fields.Str(),
+        'is_amended': ma.fields.Boolean(),
```

Comment:
Is there any reason these fields are declared here as extensions, I believe all of those should be able to be derived from the model itself, except maybe `is_amended`?